### ISSUE TITLE
Remove Favourites from the Merge file to prevent overwriting the JWL Favorites

### ISSUE THREAD

**Issue by timjobutler** [AntonyCorbett/JWLMerge#53]
### **Is your feature request related to a problem? Please describe.**
I use _JW Library_ on my Samsung phone (Android 9), Samsung tablet (Android 11) and laptop (Windows 10). On each device, I have configured a different set of Favourites. I have found that once the _JWL_ backup files are in _JWL Merge_, using the **Remove Favourites** options worked well (see screenshot below).
![JWL_Remove-Favourites](https://user-images.githubusercontent.com/37101212/130559217-a0236732-36e9-4250-a0b9-a820818a83ed.png)
However, because the Favourites section of the resulting merge file is blank, when I import the merged file into the _JWL_ app, my Favourites configuration is wiped out.

### **Describe the solution you'd like**
Add a setting to _JWL Merge_ to remove the Favourites section from the resulting merged file to preventing overwriting the Favourites configuration within the _JWL_ app.

### **Describe alternatives you've considered**
Any other option to prevent the merge file overwriting the Favourites configuration upon restoring the merge file

### **Additional context**
If I do not use the **Remove Favourites** function, all the Favourites get merged and I end up with a colossal collection of Favourites across all devices. This undoes the work that I have gone to individualise the Favourites section of the app per device.

**Comment by AntonyCorbett** [comment]
@timjobutler When JWL restores a backup file it entirely overwrites its database.

**Comment by timjobutler** [comment]
Is there any method to remove the Favourites element from the database of each backup file prior to merging them together? I either suffer the same problem as described as in issues #21 and #33 or have my Favourite mirrored on all of my devices.

Happy to explore a solution that I solely implement if it is not possible to achieve the desired results in _JWL Merge_.

Thanks Antony

**Comment by AntonyCorbett** [comment]
> Is there any method to remove the Favourites element from the database of each backup file prior to merging them together?

@timjobutler Yes, that's exactly what the **Remove Favourties** command does.

**Comment by timjobutler** [comment]
As mentioned in my OP, I use the **Remove Favourites** function **_prior_** to merging the files together.

However, since this action has left the **Favourites** section of the database empty (apologies for my poor technical explanation), it overwrites the existing **Favourites** in the app. Is there a way include the **Favourites** back into the merged file?

**Comment by AntonyCorbett** [comment]
The **Remove Favourites** command only removes the favourites from the selected backup file. So, if you have 2 backups and remove the favourites from one of them, then the favourites from the other backup will be included in the resulting merge.
